src/openrct2/ride/TrackData.cpp

```diff
@@ -1638,6 +1639,7 @@ static constexpr const rct_preview_track TrackBlocks086[] = {
     TRACK_BLOCK_END
 };
 
+// Helix Up Left Small
```

Comment:
Rather than adding comments, it is better to rename these blocks to something meaningful.

Reply:
But again leave for a different pr

Reply:
That's a good point. I'm planning to do some refactoring of the Track code, starting with documentation - would you prefer I remove these comments from this PR and do a PR later that renames these blocks?

Reply:
I would, yes.